download/_posts/2016-04-11-2.12.0-M4.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Scala 2.12.0-M4
+start: 11 April 2016
+layout: downloadpage
+release_version: 2.12.0-M4
+release_date: "April 11, 2016"
```

Comment:
I don't mind either way. I was just making sure this was intentional.

The Announce post should definitely be April 11. The Download page, IMO, should reflect the tag date, but as I said, I don't mind.